### PR TITLE
Adds CFP and financial aid reminder posts, and blog image folder

### PR DESCRIPTION
**Pull request by rebkin05**
Two last-minute reminder posts for CFP and financial aid. 

I used "image" in the front matter, but I can't tell if that worked locally... I also created a blog folder in static/img for pablo-ed images. If the images are too big, they can be made smaller, but I'm not totally sure how to do that.

### CODE REVIEW

File: _posts/2017-04-10-financial-aid-app-closes-today.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+author: Rebecca Kindschi
+category: General
+date: 2017-04-10 00:00:01
+layout: post
+image: static/img/blog/financial_aid.png
```
Comment (jefftriplett) [inline _posts/2017-04-10-financial-aid-app-closes-today.md]: Please add a `/` to the front and that should do it.

File: _posts/2017-04-10-last-call-for-cfp.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+author: Rebecca Kindschi
+category: General
+date: 2017-04-10 00:00:00
+layout: post
+image: static/img/blog/proposal_today.png
```
Comment (jefftriplett) [inline _posts/2017-04-10-last-call-for-cfp.md]: Ditto, please add a `/` to the front.

**Review by jefftriplett — CHANGES_REQUESTED** [review]

**Comment by jefftriplett** [comment]
@rebkin05 you are 👍  to go after that `/` update. I verified your images and they look great!

**Comment by williln** [comment]
@rebkin05 The images and copy are great!

File: _posts/2017-04-10-last-call-for-cfp.md (hunk 2 above)
Comment (rebkin05) [reply _posts/2017-04-10-last-call-for-cfp.md]: d'oh--thanks!

**Review by jefftriplett — APPROVED** [review]

**Comment by jefftriplett** [comment]
@rebkin05 looks good. Feel free to merge it when you are ready for it to go live.

**Comment by rebkin05** [comment]
thanks, @jefftriplett and @williln!

**Pull request merged**